components/common/src/lib.rs

```diff
@@ -52,3 +59,99 @@ lazy_static::lazy_static! {
         }
     };
 }
+
+// TODO (CM): It would be nice to come up with a way to more
+// programmatically manage these flags. It's a bit of a pain to define
+// the flag, and then define the environment variables
+// separately. Nothing statically guarantees that you've specified an
+// variable for a flag.
+
+// TODO (CM): It'd be great to have a built-in way to document them,
+// too.
+
+// TODO (CM): Part of that documentation might be *when* a flag was
+// added. In general, long-lived flags are a code-smell.
+
+// TODO (CM): It may also be useful to break out features by area of
+// concern. We can have any number of bitflags-generated structs.
+
+bitflags::bitflags! {
```

Comment:
Not to go into bike shedding territory, but are we gaining anything by using bit flags? Could we just use a plain map and not have to worry about coming up with a unique bit so specify each feature flag?

Reply:
I haven't really dug into getting off `bitflags`. However, given all the `TODO`s I dropped in here, I think there's plenty of room for future refactoring; with that, we very well may leave `bitflags` behind, as well.